app/importers/envirotech/envirotech_toolkit_data.rb

```diff
@@ -0,0 +1,29 @@
+module Envirotech
+  class EnvirotechToolkitData
```

Comment:
Isn't putting "Envirotech" in the class name redundant given that it's contained in the Envirotech module?